.globalconfig

```diff
@@ -1834,7 +1834,7 @@ dotnet_diagnostic.SA1205.severity = warning
 
 # SA1206: Declaration keywords should follow order
 # https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1206.md
-dotnet_diagnostic.SA1206.severity = none
+dotnet_diagnostic.SA1206.severity = warning
```

Comment:
I suggest migrate all our config options from the file to .editorconfig (and remove .globalconfig at all after we finish the migration).

Reply:
I prefer to follow dotnet/runtime and keep analyzer configuration in a separate `.globalconfig`.

Reply:
To follow latest `.globalconfig` from .Net SDK we have to remove our custom one and migrate our configs to .editorconfig

Reply:
> To follow latest `.globalconfig` from .Net SDK we have to remove our custom one and migrate our configs to .editorconfig

@iSazonov I do not understand why this is necessary. Can you open an issue to discuss this please?

Reply:
I add a reference to docs in your issue.

Reply:
Can we continue with this PR and then discuss analyzer configuration in https://github.com/PowerShell/PowerShell/issues/24738.